### ISSUE TITLE
strtotime is not safe when not in an english site

### ISSUE THREAD

**Issue by brunocantuaria** [WebDevStudios/CMB2#540]
Per the description on strtotime function:

"The function expects to be given a string containing an English date format..." http://php.net/manual/en/function.strtotime.php

So, those are the expected formats: http://www.gnu.org/software/shishi/manual/html_node/Date-input-formats.html

The issue is that the field 'text_datetime_timestamp' will not work as expected because of the use of 'strtotime' in regions which does not use a recognized time string. For instance, in Brazil the default use of date string is: 'DD/MM/YYYY', so our christmas date is 25/12/2015 instead of 12/25/2015.

Basically, any date after the 12th day of a month will break this field in this case (it will not save, since will return false on the 'strtotime' function).

I guess the change should be done on the javascript behavior, right? So regardless of the language in the input, it will store the date as english datetime.

**Comment by jtsternberg** [comment]
Could you please test the [trunk branch](https://github.com/WebDevStudios/CMB2/tree/trunk) as there have been a lot of date issues addressed there.

**Comment by brunocantuaria** [comment]
Ok, it's working with the trunk ;)

**Comment by jtsternberg** [comment]
:+1:
